altinn/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/altinn/App.kt

```diff
@@ -25,11 +28,29 @@ fun RapidsConnection.createAltinn(altinnClient: AltinnClient): RapidsConnection
         AltinnRiver(altinnClient).connect(this)
     }
 
-private fun createAltinnClient(): AltinnClient =
-    AltinnClient(
+private fun createAltinnClient(): AltinnClient {
+    val maskinportenClient: MaskinportenClient = createMaskinportenClient()
```

Comment:
Vi definerer vanligvis ikke typen på variabler i Kotlin. Unntaket er "platform types", aka Java-typer som må defineres for å si noe om nullability.